Assignment6.Rmd

```diff
@@ -23,19 +25,22 @@ library(rpart)
 #Upload the data sets MOOC1.csv and MOOC2.csv
 M1 <- read.csv("MOOC1.csv", header = TRUE)
 
-M2 <- 
+M2 <- read.csv("MOOC2.csv", header = TRUE)
 
 ```
 
 #Decision tree
 ```{r}
 #Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
 
-c.tree1 <- 
+##Answer: I would assume that those who get higher grades in final exam and assignments would have higher probabilities of getting certified. So I would choose "grade + assignment" 2 variables in this case.
+##Note: previously, when choosing all three variables, only forum.posts are unsed by R in tree construction.
 
```

Comment:
You may find this is because the variables have not been scaled and `forum.posts` has many more increments than the other variables.